bootstrap/peer.go

```diff
@@ -19,6 +19,7 @@ import (
 	"storj.io/storj/pkg/kademlia"
 	"storj.io/storj/pkg/pb"
 	"storj.io/storj/pkg/peertls/tlsopts"
+	"storj.io/storj/pkg/revocation"
```

Comment:
Wondering whether there's a better place than the `pkg` dir.

Maybe `pkg/identity/revocation`, `pkg/peertls/revocation` or `pkg/peertls/tlsopts/revocation`.

Reply:
It seems peertls/extensions contains a bunch of additional revocation logic... so maybe

1. keep interface for database in `peertls/revocation`,
2. move revocation logic handling from `peertls/extensions` into `peertls/revocation`
3. keep implementation in `peertls/revocation/revocationkv`, and let the caller pass in the right interface

But, these can be done in separate PR-s.